packages/site/src/ui/example/react.js

```diff
@@ -234,10 +234,12 @@ class ReactExample extends React.Component {
         {this.renderError()}
 
         {this.props.themeToggle && (
-          <ThemeToggle
-            activeThemeName={this.state.themeName}
-            onSelect={this.handleThemeSelect}
-          />
+          <Theme name={this.state.themeName}>
```

Comment:
The solution doesn't quite make sense to me.

`state.themeName` is the internal state for the theme toggling.  But it should change the *look* of the theme toggler.  Right?

Reply:
I talked through the solution with Edward, and this solution now makes sense.  ViewToggle didn't used to consume Theme.  Now it does.  Through Context.  Page-level context is light theme.  That could be the reason that the inactive option is too dark.  

Edward is double-checking that all commits are in and works on his box.